simulator/map_to_schema.py

```diff
@@ -0,0 +1,102 @@
+def MapVMSampleToSchema(VMSample):
+
+    if VMSample:
+
+        VMInfo = {
```

Comment:
VMInfo is a variable name, not a type. You should follow the standard: https://google.github.io/styleguide/pyguide.html#316-naming

Same for other variable names.

Reply:
updated